src/core/command.cpp

```diff
@@ -312,11 +313,17 @@ bool Command::set_attribute(JsonObject output, const char * cmd, const char * at
         return true;
     }
 
-    // not found
     output.clear();
 
+    // attribute isn't found
+    // it could be a value command, but the value doesn't exist?
+    if (strcmp(attribute, "value") == 0) {
+        logger_.debug("%s has no value set", cmd);
```

Comment:
better use `LOG_DEBUG` as all other loggings uses the macro.

Reply:
good one, I'll change.